TheFoodieNetwork/app/src/main/java/com/fbu/thefoodienetwork/activities/SearchActivity.java

```diff
@@ -1,31 +1,42 @@
 package com.fbu.thefoodienetwork.activities;
 
+import android.Manifest;
+import android.content.Context;
 import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.location.LocationManager;
 import android.os.Bundle;
 import android.os.Handler;
 import android.util.Log;
 import android.view.KeyEvent;
 import android.view.View;
 import android.view.inputmethod.EditorInfo;
 import android.widget.TextView;
+import android.widget.Toast;
 
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.app.ActivityCompat;
 import androidx.recyclerview.widget.LinearLayoutManager;
 
-import com.fbu.thefoodienetwork.API_Severs.ZomatoRequest;
 import com.fbu.thefoodienetwork.adapters.LocationAdapter;
 import com.fbu.thefoodienetwork.adapters.RestaurantAdapter;
+import com.fbu.thefoodienetwork.apiservers.ZomatoRequest;
 import com.fbu.thefoodienetwork.databinding.ActivitySearchBinding;
+import com.fbu.thefoodienetwork.keys.ParcelKeys;
 import com.fbu.thefoodienetwork.models.Location;
 import com.fbu.thefoodienetwork.models.Restaurant;
 
 import org.parceler.Parcels;
 
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
+//TODO progress bar for waiting for network calls
 public class SearchActivity extends AppCompatActivity implements LocationAdapter.OnClickLocationListener, RestaurantAdapter.OnClickRestaurantListener {
     private static final String TAG = "SearchActivity";
+    private static final int REQUEST_LOCATION = 100;
```

Comment:
Yes, Love to see this as a constant!  I'm wondering if there's a more descriptive name, though 🤔